docs/topics/certificaterotation.md

```diff
@@ -0,0 +1,52 @@
+# Cluster certificate rotation
+
+Instructions on rotating TLS CA and certificates for an AKS Engine cluster.
+
+## Prerequesites
+
+- Kubernetes version 1.8.0 or later is required
+- The etcd members MUST be in a healthy state before rotating the CA and certs.
+- The `_output` directory for your cluster deployment, including a working SSH key for accessing the master nodes and `kubeconfig`.
+
+## Preparation
+
+**CAUTION**: Before rotation, it's preferrable to back up your cluster:
+
+1. On an etcd node, take a backup of the current state.
+2. //TODO recommend etcd backup tool, extract the existing state from etcd. Copy this back to your working machine as a precaution, in case the control plane goes down.
+3. Keep a copy of your current _output directory.
+
+## Rotation
+
+**CAUTION**: Rotating certificates will cause cluster downtime.
+
+run `aks-engine rotate-certs`
+
+`aks-engine rotate-certs` will:
+
+- Use the exisiting certificate generation code to generate new certificates, using the same certificate properties and default expiration as new certificates generated for new clusters.
```

Comment:
Do we need to mention "Use the existing certificate genaration code" here? I think "Generate new certificates" is good enough.

Reply:
Yeah I'll remove that, that was mostly when I was trying to make this a design doc by describing the implementation.